includes/class-wc-payments-order-service.php

```diff
@@ -794,4 +795,52 @@ private function update_order_status( $order, $order_status, $intent_id = '' ) {
 			Logger::log( 'Error when updating status for order ' . $order->get_id() . ': ' . $e->getMessage() );
 		}
 	}
+
+	/**
+	 * Return terminal intent payment method array based on payment methods request.
+	 *
+	 * @param WP_REST_Request $request Request object.
+	 * @param array           $default_value - default value.
+	 *
+	 * @return array|null
+	 * @throws \Exception
+	 */
+	public function get_terminal_intent_payment_method( $request, array $default_value = [ Payment_Method::CARD_PRESENT ] ) :array {
+		if ( null === $request->get_param( 'payment_methods' ) ) {
```

Comment:
I'd suggest to introduce a local variable for value of `$request->get_param( 'payment_methods' )`.